### ISSUE TITLE
IOException in :0

### ISSUE THREAD

**Issue by mbos** [mbos/Pass4Win#67]
## Exception in Pass4Win

**IOException** in **:0**
The process cannot access the file &#x27;C:\Users\rilott\AppData\Local\Pass4Win\config.json&#x27; because it is being used by another process.

[View on Bugsnag](https://bugsnag.com/pass4win/pass4win/errors/561b730122e621496551043b?event_id=561b730131feec0a65946e50)
## Stacktrace

```
:0 - System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
:0 - System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
:0 - System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
```

[View full stacktrace](https://bugsnag.com/pass4win/pass4win/errors/561b730122e621496551043b?event_id=561b730131feec0a65946e50)

**Comment by mbos** [comment]
Implement a check that only one instance of the application is running

**Comment by mbos** [comment]
Solved in commit 9129baf
